src/soundio/sounddeviceportaudio.h

```diff
@@ -26,17 +26,17 @@
 #include "soundio/sounddevice.h"
 #include "util/duration.h"
 #include "util/fifo.h"
+#include "util/logging.h"
+
+Q_DECLARE_LOGGING_CATEGORY(mixxxLogDevicePortAudio)
+#define MIXXX_LOGGING_CATEGORY_DEVICE_PORTAUDIO \
+MIXXX_LOGGING_CATEGORY_PARENT_CHILD(MIXXX_LOGGING_CATEGORY_DEVICE, "PortAudio")
 
 #define CPU_USAGE_UPDATE_RATE 30 // in 1/s, fits to display frame rate
 
 class SoundManager;
 class ControlProxy;
 
-/** Dynamically resolved function which allows us to enable a realtime-priority callback
-    thread from ALSA/PortAudio. This must be dynamically resolved because PortAudio can't
-    tell us if ALSA is compiled into it or not. */
-typedef int (*EnableAlsaRT)(PaStream* s, int enable);
```

Comment:
Is this unused? Why did you change it in this logging PR?

Reply:
Moved to the .cpp file, not needed in the header.